web/gridt/tests/integration/resources/test_leader.py

```diff
@@ -11,44 +11,60 @@
 
 
 class SwapTest(BaseTest):
-    def test_swap_leader(self):
+    def test_swap_leader_correctly(self):
         with self.app_context():
-            movement = Movement("Flossing", "daily", "Hi")
-
-            user1 = User("test1", "test1@test.com", "pass")
-            user2 = User("test2", "test2@test.com", "pass")
-            user3 = User("test3", "test3@test.com", "pass")
-            user4 = User("test4", "test4@test.com", "pass")
-            user5 = User("test5", "test5@test.com", "pass")
-
-            signal = Signal(user3, movement)
-            time_stamp = signal.time_stamp
+            movement = self.create_movement()
+            user1 = self.create_user_in_movement(movement)
```

Comment:
Maybe neater in a list comprehension or for loop.

Reply:
Good idea! The users don't actually need to be named either, so a for loop is good.